README.md

```diff
@@ -101,8 +102,23 @@ The released version of the SDK can be downloaded from https://github.com/Appdyn
     * [C++ SDK and Sample App](https://github.com/Appdynamics/iot-cpp-sdk)
     * [Python Sample Code](https://github.com/Appdynamics/iot-rest-api-sample-apps)
 
+## Miscellaneous
+* Code Coverage 
+    * `jacoco` is integrated in the `build.gradle` file to track code coverage
+    * Generate report
+    ```bash
+    cd <MY_SOURCE_FOLDER>/iot-java-sdk
+    ./gradlew test jacocoTestReport
+    ```
+    * View the report in your favorite browser
```

Comment:
2. View the report by opening the file `<MY_SOURCE_FOLDER>/iot-java-sdk/sdk/build/jacocoHtml/index.html` in a browser.